tests/test_zenos.py

```
@@ -1,77 +1,81 @@
         print(f"   📊 Registry initialized: {len(registry.plugins)} plugins")
-        
+
         # Test stats
         stats = registry.get_collection_stats()
         print(f"   📈 Collection stats: {stats}")
-        
+
         print("   ✅ Plugin system working!")
-        
+
     except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)